iron-fit-behavior.html

```diff
@@ -501,29 +491,35 @@
     },
 
 
-    __getPosition: function(hAlign, vAlign, size, positionRect, fitRect) {
+    __getPosition: function(hAlign, vAlign, rect, positionRect, fitRect) {
+      var margin = this._fitInfo.margin;
+      // Consider the margin as part of the size for position calculations.
+      var size = {
```

Comment:
Yeah, this method also accesses to other properties like `dynamicAlign` or `noOverlap`. I could move also the `verticalAlign/horizontalAlign` in it and keep the signature to be `function(rect, positionRect, fitRect)`...WDYT?